Intake group name format mismatch between intake / GM module.

HarmdR opened the pull request:
finally een streep eronder 👍

HarmdR committed:
Added intake group like "intake-foo"

HarmdR committed:
wip

HarmdR committed:
wip

stsnel commented on roles/yoda_rulesets/templates/job_movetovault.r.j2:
```diff
@@ -13,22 +13,41 @@
+                    }
+                }
+                *intakeRoot = "/*zone/home/*grp_name";
+                *vaultRoot  = "/*zone/home/grp-vault-*grp";
```
This results in a rule failure on groups with an "intake-" prefix, since groups with this prefix use "vault-" rather than "grp-vault" as their prefix.

This probably also applies to the other copy of this script in the yoda_test role

stsnel commented on roles/yoda_rulesets/templates/job_movetovault.r.j2:
```diff
@@ -13,22 +13,41 @@
+                        *found = true;
+                        break;
+                    }
+                    if (*found) {
```
It would be helpful for troubleshooting purposes if there was a check here for whether the group name was found, so that the rule could log an error message about that.

stsnel committed:
Add logic to movetovault rule

- If intake group has "intake-" prefix, vault should have "vault-" prefix
- Log error in case job can't find any of the listed groups

stsnel committed:
Intake: fix log directory intake-to-vault job

So that it matches current iRODS log directory and works on
development instances

stsnel committed:
YDA-4946: extend intake timeout

If intake module is enabled, extend portal session timeouts and portal
VHost timeouts in order to accommodate long synchronous operations
such as intake scans.

stsnel committed:
test: ensure test intake group names match

stsnel committed:
test: ensure rods access to intake vaults

needed for the movetovault job to be able to run correctly
